src/views/Search.vue

```diff
@@ -37,8 +37,9 @@
 
           <ResultSection class="rf-margin-top-4N rf-padding-top-4N">
           <template v-slot:titleResultSection>
+            <p v-if="text == null">Aucune recherche n’a pu être effectuée car vous n’avez pas saisi de terme de recherche</p>
```

Comment:
Ajout du point à la fin de la phrase.

```suggestion
            <p v-if="text == null">Aucune recherche n’a pu être effectuée car vous n’avez pas saisi de terme de recherche.</p>
```